src/plugins/controls/public/time_slider/embeddable/time_slider_embeddable.tsx

```diff
@@ -354,6 +352,10 @@ export class TimeSliderControlEmbeddable extends Embeddable<
       .format(this.getState().componentState.format);
   };
 
+  public clearSelections() {
+    this.onTimesliceChange();
```

Comment:
There is something going on with Maps embeddable panels where clearing time slider does not refetch map with correct data. Instead, map still shows data from the previously selected time slice. Maybe you need to pass `[timeRangeMin, timeRangeMax]` into onTimesliceChange? Or debug the issue in map embeddable

See screen shots below

Reply:
I have found the problem in the map_embeddable. You need to pass `clearTimeslice` to `setQuery`, so change `MapEmbeddable._dispatchSetQuery` to the below implemenation.

```
_dispatchSetQuery({ forceRefresh }: { forceRefresh: boolean }) {
    this._savedMap.getStore().dispatch<any>(
      setQuery({
        filters: this._getInputFilters(),
        query: this.input.query,
        timeFilters: this.input.timeRange,
        timeslice: this.input.timeslice
          ? { from: this.input.timeslice[0], to: this.input.timeslice[1] }
          : undefined,
        clearTimeslice: this.input.timeslice === undefined,
        forceRefresh,
        searchSessionId: this._getSearchSessionId(),
        searchSessionMapBuffer: getIsRestore(this._getSearchSessionId())
          ? this.input.mapBuffer
          : undefined,
      })
    );
  }
```

Reply:
Oooh yes that makes total sense - thank you! I completely forgot to test the Maps use of the time slider, so thank you for catching that 🙇

Reply:
Fixed in b3ccbc8335a5c245e3508ecaf5e176b5cf885714 :+1: